surefire-api/src/main/java/org/apache/maven/surefire/api/testset/TestListResolver.java

```diff
@@ -135,27 +131,6 @@ public boolean hasMethodPatterns()
         return hasIncludedMethodPatterns() || hasExcludedMethodPatterns();
     }
 
-    /**
-     *
-     * @param resolver    filter possibly having method patterns
-     * @return {@code resolver} if {@link TestListResolver#hasMethodPatterns() resolver.hasMethodPatterns()}
-     * returns {@code true}; Otherwise wildcard filter {@code *.class} is returned.
-     */
-    public static TestListResolver optionallyWildcardFilter( TestListResolver resolver )
-    {
-        return resolver.hasMethodPatterns() ? resolver : WILDCARD;
-    }
-
-    public static TestListResolver getEmptyTestListResolver()
-    {
-        return EMPTY;
-    }
-
-    public final boolean isWildcard()
```

Comment:
This is still a valid case.
If TLR is empty (nothing) or a wildcard (everything), the provider would ignore TLR. But both corner cases may happen in the config.

Reply:
only one case is true for `isWildcard`:

```java
        tlr = new TestListResolver( "*.class" );
        assertTrue( tlr.isWildcard() );
```